addons/refuel/functions/fnc_connectNozzleAction.sqf

```diff
@@ -0,0 +1,109 @@
+/*
+ * Author: GitHawk et.al
+ * Calculates a connection for tilting
+ * With code from ace_attach
```

Comment:
We could probably generalize some of the attachment/placement code in future and add them to common as functions.